### DISCUSSION TITLE
Feature Request: When sharing a toot/post, have the user comment in the shared link preview

### DISCUSSION THREAD

**Discussion by ziggyb1013** [mastodon/mastodon#21535] [General]
When I share a toot on Messages on iOS, it shows the preview images and user info, but NOT the text of the post. That's included as a second "message."

When sharing a similar tweet, it's all nice and tidy as one link preview including the post and user info. 

Below is a sample of what happens. 
1. Toot/Post on Mastodon
2. What it looks like when shared via Messages on iOS
3. When a similar tweet is shared

![8039FD44-6BA6-44A5-A6A8-70AFC7FFCE7A](https://user-images.githubusercontent.com/111400148/203679611-690621b9-f084-4042-815c-1c40caf3cfbd.jpeg)
![A9887A79-85D2-4E10-BC61-661E5BD23CA0](https://user-images.githubusercontent.com/111400148/203679612-c1cd4d3b-5452-4451-b333-44775e143b34.jpeg)
![693A77FC-7C0A-48B2-91E2-8E87523294D6](https://user-images.githubusercontent.com/111400148/203679615-2a7e1025-340c-4465-acbf-c3f26aaf8164.jpeg)

**Comment by jaanus** [comment]
Yep. I reported the same in the issue tracker: https://github.com/mastodon/mastodon/issues/20569
